web/src/plugins/pipelines/useDnD.ts

```
@@ -0,0 +1,644 @@
+  
+  
+ 
+  const filteredOptions = ref(formattedOptions.value);
+
+  const filterOptions = (val:any, update:any) => {
+    if (val === '') {
+      filteredOptions.value = formattedOptions.value;
+    } else {
+      const filterFn = (option:any) =>
+        option.label.toLowerCase().includes(val.toLowerCase()) ||
+        (option.node_type && option.node_type.toLowerCase().includes(val.toLowerCase()));
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)